kolibri/plugins/learn/assets/src/views/explore-page/index.vue

```diff
@@ -13,11 +13,18 @@
       {{ topic.description }}
     </p>
 
+    <ui-select
+      :label="$tr('display')"
+      :options="filterOptions"
+      v-model="selectedFilter"
+      class="filter"
+    />
+
     <span class="visuallyhidden" v-if="subtopics.length">{{ $tr('navigate') }}</span>
 
-    <card-grid v-if="contents.length">
+    <card-grid v-if="filteredContents.length">
       <content-grid-item
-        v-for="content in contents"
+        v-for="content in filteredContents"
```

Comment:
Off the top of my head, something like this might work:

```
<content-grid-item v-show="selectedFilter ==='all' ||content.kind === selectedFilter.value ">
```